### DISCUSSION TITLE
Backup failed because The dump process failed with exitcode 1 : General error : The system cannot find the path specified.

### DISCUSSION THREAD

**Discussion by tamim-ikbal** [spatie/laravel-backup#1263] [Q&A]
Backup failed because The dump process failed with exitcode 1 : General error : The system cannot find the path specified.
.
#0 E:\xampp\htdocs\larastarter\vendor\spatie\db-dumper\src\DbDumper.php(263): Spatie\DbDumper\Exceptions\DumpFailed::processDidNotEndSuccessfully(Object(Symfony\Component\Process\Process))
#1 E:\xampp\htdocs\larastarter\vendor\spatie\db-dumper\src\Databases\MySql.php(203): Spatie\DbDumper\DbDumper->checkIfDumpWasSuccessFul(Object(Symfony\Component\Process\Process), 'E:\\xampp\\htdocs...')
#2 E:\xampp\htdocs\larastarter\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(244): Spatie\DbDumper\Databases\MySql->dumpToFile('E:\\xampp\\htdocs...')
#3 [internal function]: Spatie\Backup\Tasks\Backup\BackupJob->Spatie\Backup\Tasks\Backup\{closure}(Object(Spatie\DbDumper\Databases\MySql), 'mysql')
#4 E:\xampp\htdocs\larastarter\vendor\laravel\framework\src\Illuminate\Collections\Collection.php(640): array_map(Object(Closure), Array, Array)
#5 E:\xampp\htdocs\larastarter\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(247): Illuminate\Support\Collection->map(Object(Closure))
#6 E:\xampp\htdocs\larastarter\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(164): Spatie\Backup\Tasks\Backup\BackupJob->dumpDatabases()
#7 E:\xampp\htdocs\larastarter\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(140): Spatie\Backup\Tasks\Backup\BackupJob->createBackupManifest()
#8 E:\xampp\htdocs\larastarter\vendor\spatie\laravel-backup\src\Commands\BackupCommand.php(54): Spatie\Backup\Tasks\Backup\BackupJob->run()
#9 E:\xampp\htdocs\larastarter\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(36): Spatie\Backup\Commands\BackupCommand->handle()
#10 E:\xampp\htdocs\larastarter\vendor\laravel\framework\src\Illuminate\Container\Util.php(40): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#11 E:\xampp\htdocs\larastarter\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(93): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#12 E:\xampp\htdocs\larastarter\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(37): Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Array, Object(Closure))
#13 E:\xampp\htdocs\larastarter\vendor\laravel\framework\src\Illuminate\Container\Container.php(610): Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), Array, Array, NULL)
#14 E:\xampp\htdocs\larastarter\vendor\laravel\framework\src\Illuminate\Console\Command.php(136): Illuminate\Container\Container->call(Array)
#15 E:\xampp\htdocs\larastarter\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
#16 E:\xampp\htdocs\larastarter\vendor\laravel\framework\src\Illuminate\Console\Command.php(121): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
#17 E:\xampp\htdocs\larastarter\vendor\spatie\laravel-backup\src\Commands\BaseCommand.php(16): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 E:\xampp\htdocs\larastarter\vendor\symfony\console\Application.php(971): Spatie\Backup\Commands\BaseCommand->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 E:\xampp\htdocs\larastarter\vendor\symfony\console\Application.php(290): Symfony\Component\Console\Application->doRunCommand(Object(Spatie\Backup\Commands\BackupCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 E:\xampp\htdocs\larastarter\vendor\symfony\console\Application.php(166): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 E:\xampp\htdocs\larastarter\vendor\laravel\framework\src\Illuminate\Console\Application.php(92): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 E:\xampp\htdocs\larastarter\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(129): Illuminate\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#23 E:\xampp\htdocs\larastarter\artisan(37): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#24 {main}
Backup failed because: The dump process failed with exitcode 1 : General error : The system cannot find the path specified.

mysql config

'mysql' => [
            'driver' => 'mysql',
            'url' => env('DATABASE_URL'),
            'host' => env('DB_HOST', '127.0.0.1'),
            'port' => env('DB_PORT', '3306'),
            'database' => env('DB_DATABASE', 'forge'),
            'username' => env('DB_USERNAME', 'forge'),
            'password' => env('DB_PASSWORD', ''),
            'unix_socket' => env('DB_SOCKET', ''),
            'charset' => 'utf8mb4',
            'collation' => 'utf8mb4_unicode_ci',
            'prefix' => '',
            'prefix_indexes' => true,
            'strict' => true,
            'engine' => null,
            'options' => extension_loaded('pdo_mysql') ? array_filter([
                PDO::MYSQL_ATTR_SSL_CA => env('MYSQL_ATTR_SSL_CA'),
            ]) : [],
            'dump' => [
                'dump_binary_path' => '/path/to/the/binary', // only the path, so without `mysqldump` or `pg_dump`
                'use_single_transaction',
                'timeout' => 60 * 5, // 5 minute timeout
                'add_extra_option' => '--optionname=optionvalue', // for example '--column_statistics=0'
            ],
        ],

**Comment by tamim-ikbal** [comment]
How can I solved this?

**Reply by shanmugarajah25** [reply]
@Tamim26061
try this. it's working for me
 `'dump' => [
                'dump_binary_path' => 'C:\xampp\mysql\bin',
                'use_single_transaction',
                'timeout' => 60 * 5, // 5 minute timeout
                //  'exclude_tables' =>['table1', 'table2'],
                //'add_extra_option' =>'--optionname=optionvalue',
             ]`

**Comment by chourik2000-prog** [comment]
j'ai le même problème

**Reply by chourik2000-prog** [reply]
Dans mon fichier database.php , je retrouve pas 'dump'=> ...

**Reply by fredsal** [reply]
I don't understand, which language is that?
